spec/menu_spec.rb

```diff
@@ -0,0 +1,35 @@
+require_relative '../app/lib/menu'
+
+describe 'a menu' do
+
+  let(:dummy_receipt) { Class.new { extend Menu } }
```

Comment:
This is a clever way of being able to test a module but this is one of the reasons I prefer objects over modules - they're easier to test.

That said when I do test modules I prefer to use [rspec shared examples](https://www.relishapp.com/rspec/rspec-core/v/2-11/docs/example-groups/shared-examples) which I then include in the specs that use the module